pyannote/audio/labeling/tasks/speech_activity_detection.py

```diff
@@ -304,7 +306,19 @@ def batch_loss(self, batch):
 
 
 class DomainAdversarialSpeechActivityDetection(DomainAwareSpeechActivityDetection):
-    """TODO: docstring"""
+   """Domain Adversarial speech activity detection
```

Comment:
Looks like we are missing one space here.

Reply:
Oops, missed that! It's fixed now